src/main/resources/org/sagebionetworks/web/client/widget/entity/editor/TeamSelectEditorViewImpl.ui.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
+	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html">
+	
+	<bh:Div>
+		<bh:Div addStyleNames="form-horizontal">
+			<b:FieldSet>
+				<b:FormGroup>
+					<b:FormLabel>Team Name</b:FormLabel>
```

Comment:
Here's the Team Name label!

Reply:
The UI is still not great, but now it has a label:
<img width="1299" alt="Screen Shot 2021-03-31 at 10 41 41 AM" src="https://user-images.githubusercontent.com/1864447/113187536-d3c40080-920d-11eb-8262-bbe8386229e6.png">